src/FilesystemPool.php

```
@@ -186,7 +186,7 @@ public function save(CacheItemInterface $item)
 
         $data = serialize($item);
 
-        $result = file_put_contents($this->getPath($key), $data);
+        $result = @file_put_contents($this->getPath($key), $data);
```

Please don't use the `@` error suppressor, as it hides all errors it can cause bugs to go unnoticed

---

If we use the ```error suppressor```, it can let ```$result``` be ```false``` and return value.
And it cannot throw the related ```Exception``` directly in executing ```file_put_contents```.

We should figure out another way to handle this. Otherwise the line ```191``` will not reached.

---

I remove the line ```191``` if condition block because this block I think it's not reached.

---

It will be reached, `file_put_contents()` returns `false` on failure, see http://php.net/manual/en/function.file-put-contents.php